cli/src/benchmark_gpt.ts

```diff
@@ -80,10 +80,10 @@ async function main(args: Required<CLIArguments>): Promise<void> {
     console.log(`\tmodel type ${modelType} \n\tbatch size ${batchSize} \n\tcontext length ${contextLength}`)
 
     let epochTime = performance.now()
-    const logGenerator = model.train(preprocessedDataset, undefined, epoch)
-    for await (const logs of logGenerator) {
+    for (let epochsCounter = 0; epochsCounter < epochsCount; epochsCounter++) {
```

Comment:
```suggestion
    for (let epochsCounter = 1; epochsCounter <= epochsCount; epochsCounter++) {
```

There is a division by 0 for the first `msPerToken` if `epochCounter` starts at 0